docs/docs/developer-guide.md

```diff
@@ -25,17 +21,15 @@ echo 'export ICEBERG_REST__PG_DATABASE_URL_READ="postgresql://postgres:postgres@
 echo 'export ICEBERG_REST__PG_DATABASE_URL_WRITE="postgresql://postgres:postgres@localhost/postgres"' >> .env
 source .env
 
-# migrate db
-cd crates/lakekeeper
-sqlx database create && sqlx migrate run
-cd ../..
-
```

Comment:
This needs to be done on initial setup to add our tables etc, otherwise the db is empty. So I think these commands should remain in that section.